src/ispc.cpp

```diff
@@ -480,6 +486,8 @@ class AllCPUs {
 #if ISPC_LLVM_VERSION >= ISPC_LLVM_12_0
         names[CPU_AppleA14].push_back("apple-a14");
 #endif
+        names[CPU_MicrosoftSQ1].push_back("microsoft-sq1");
```

Comment:
Neither `microsoft-sq1`, nor `microsoft-sq2` are available as CPU definitions in open source LLVM. They should likely to be defined somewhere in [llvm/lib/Target/AArch64/AArch64.td](https://github.com/llvm/llvm-project/blob/main/llvm/lib/Target/AArch64/AArch64.td).

Would it be sufficient to use just default CPU definition for aarch64?